lightspeed_restaurant.gemspec

```diff
@@ -12,16 +12,19 @@ Gem::Specification.new do |spec|
 
   spec.summary       = 'Ruby bindings for the Lightspeed Restaurant API'
   spec.description   = 'Lightspeed Restaurant is a Point Of Sale that helps bars, restaurants, '\
-                       'and cafés deliver a better customer experience and run a more profitable business. '\
-                       'See https://www.lightspeedhq.com/products/restaurant/ for details'
+                       'and cafés deliver a better customer experience and run a more profitable '\
+                       'business. See https://www.lightspeedhq.com/products/restaurant/ for details'
   spec.homepage      = 'https://github.com/chronogolf/lightspeed_restaurant'
   spec.license       = 'MIT'
 
-  spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
+  spec.files         =
+    `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
   spec.bindir        = 'exe'
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
+  spec.required_ruby_version = '>= 2.6.5'
```

Comment:
I had to put a ruby version here to comply with `RuboCop::Cop::Gemspec::RequiredRubyVersion` rule. I used the version specified in `.ruby-version` but maybe the version could be lower 🤷